File: android/versioned-abis/expoview-abi42_0_0/src/main/java/abi42_0_0/expo/modules/constants/ConstantsPackage.java
package abi42_0_0.expo.modules.constants;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi42_0_0.org.unimodules.core.ExportedModule;
import abi42_0_0.org.unimodules.core.BasePackage;
import abi42_0_0.org.unimodules.core.interfaces.InternalModule;

public class ConstantsPackage extends BasePackage {
  @Override
  public List<InternalModule> createInternalModules(Context context) {
    return Collections.singletonList((InternalModule) new ConstantsService(context));
  }

  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new ConstantsModule(context));
  }
}

File: android/versioned-abis/expoview-abi44_0_0/src/main/java/abi44_0_0/expo/modules/constants/ConstantsPackage.kt
package abi44_0_0.expo.modules.constants

import android.content.Context

import abi44_0_0.expo.modules.core.BasePackage
import abi44_0_0.expo.modules.core.interfaces.InternalModule
import abi44_0_0.expo.modules.core.ExportedModule

class ConstantsPackage : BasePackage() {
  override fun createInternalModules(context: Context): List<InternalModule> =
    listOf(ConstantsService(context))

  override fun createExportedModules(context: Context): List<ExportedModule> =
    listOf(ConstantsModule(context))
}
